From fafd610db3447288f8c4ab8478c418b12d03bf62 Mon Sep 17 00:00:00 2001
From: Vinay Simha BN <vinaysimha@inforcecomputing.com>
Date: Thu, 10 Dec 2015 18:51:51 +0530
Subject: [PATCH 04/17] leds: user-configurable leds for ifc6309

DS6 - LED1 - APQ  - GPIO_95  - (mmc0)
DS5 - LED2 - APQ  - GPIO_120 - (mmc1)
DS1 - LED3 - PMIC - GPIO_01  - (heartbeat)
DS4 - LED4 - PMIC - GPIO_02  - (default-on)

Signed-off-by: Vinay Simha BN <vinaysimha@inforcecomputing.com>
---
 arch/arm64/boot/dts/qcom/apq8016-sbc.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/apq8016-sbc.dtsi b/arch/arm64/boot/dts/qcom/apq8016-sbc.dtsi
index b9d2caf..cfcbfee 100644
--- a/arch/arm64/boot/dts/qcom/apq8016-sbc.dtsi
+++ b/arch/arm64/boot/dts/qcom/apq8016-sbc.dtsi
@@ -220,30 +220,30 @@
 
 			led@1 {
 				label = "apq8016-sbc:green:user1";
-				gpios = <&msmgpio 21 GPIO_ACTIVE_HIGH>;
+				gpios = <&msmgpio 95 GPIO_ACTIVE_HIGH>;
 				linux,default-trigger = "heartbeat";
-				default-state = "off";
+				default-state = "mmc0";
 			};
 
 			led@2 {
 				label = "apq8016-sbc:green:user2";
 				gpios = <&msmgpio 120 GPIO_ACTIVE_HIGH>;
-				linux,default-trigger = "mmc0";
+				linux,default-trigger = "mmc1";
 				default-state = "off";
 			};
 
 			led@3 {
 				label = "apq8016-sbc:green:user3";
 				gpios = <&pm8916_gpios 1 GPIO_ACTIVE_HIGH>;
-				linux,default-trigger = "mmc1";
+				linux,default-trigger = "heartbeat";
 				default-state = "off";
 			};
 
 			led@4 {
 				label = "apq8016-sbc:green:user4";
 				gpios = <&pm8916_gpios 2 GPIO_ACTIVE_HIGH>;
-				linux,default-trigger = "none";
-				default-state = "off";
+				linux,default-trigger = "default-on";
+				default-state = "on";
 			};
 
 			led@5 {
-- 
1.9.1

